cmake/morpheus_utils/python/python_module_tools.cmake

```diff
@@ -622,10 +622,11 @@ macro(__create_python_library MODULE_NAME)
     # Before installing, create the custom command to generate the stubs
     add_custom_command(
       OUTPUT  ${module_binary_stub_file}
-      COMMAND ${Python3_EXECUTABLE} -m pybind11_stubgen ${TARGET_NAME} --no-setup-py --log-level WARN -o ./ --root-module-suffix \"\"
+      COMMAND env PYTHONPATH=${module_root_binary_dir}:$PYTHONPATH ${Python3_EXECUTABLE} -m pybind11_stubgen ${TARGET_NAME} --no-setup-py --log-level WARN -o ./ --root-module-suffix ""
```

Comment:
I think using the CMake command `-E` is preferred here. Something like `add_custom_target(newtarget ${CMAKE_COMMAND} -E env NAME=VALUE somecommand)`

Reply:
`add_custom_target` has the following behavior:
> The target has no output file and is always considered out of date.

I don't think we want this which is why we have `add_custom_command` generate the stub file since we can easily specify the output file.